File: exercicios/R/1158/1158.r
input = file('stdin','r')

n = as.integer(readLines(input,n=1))

for(i in 1:n){
    soma = as.integer(0)
    j = as.integer(1)
    line = strsplit(readLines(input,n=1), " ")

    x = as.integer(line[[1]][1])
    y = as.integer(line[[1]][2])

    while(j<=y){
        if(x%%2!=0){
            soma = soma + x

            x = x +1
            j = j + 1
        }else if(x%%2 == 0){
            x = x + 1
        }
        
    }
    write(soma,'')
}

File: exercicios/JavaScript/1158/1158.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift());

for(let i=0; i<n; i++){
    var line1 = lines.shift().split(' ');

    var x = parseInt(line1[0]), y = parseInt(line1[1]),j=1,soma=0;

    while(j<=y){
        if(x%2 !=0){
            soma = soma + x;

            x = x + 1;
            j = j +1;
        }
        if(x%2==0){
            x = x + 1;
        }
    }
    console.log(soma);
}